guides/md/keras_nlp/transformer_pretraining.md

```diff
@@ -146,6 +159,9 @@ print(sst_train_ds.unbatch().batch(4).take(1).get_single_element())
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:From /opt/conda/envs/myenv/lib/python3.9/site-packages/tensorflow/python/autograph/pyct/static_analysis/liveness.py:83: Analyzer.lamba_check (from tensorflow.python.autograph.pyct.static_analysis.liveness) is deprecated and will be removed after 2023-09-23.
```

Comment:
Can we remove these three lines, this should be fixed in upstream tensorflow shortly, and is just distracting for now.